### DISCUSSION TITLE
Minimum Edge Version

### DISCUSSION THREAD

**Discussion by ivanjx** [MicrosoftEdge/WebView2Feedback#2198] [Q&A] [answered]
What is the minimum version of ms edge to get the webview evergreen automatically without installing it separately? Thanks.

**Comment by victorthoang** [comment, marked as answer]
@realivanjx ,

WebView2 works with Edge version 86 and higher. If you also have Windows 11, it should be automatically installed with Evergreen. If you're using Windows 10, there's currently an aggressive roll out to have WebView 2, on the OS as a windows update.
